src/python_testing/TC_PAVSTI_Utils.py

```diff
@@ -150,11 +150,8 @@ def assert_valid_ccdid(self, ccdid: int) -> None:
         asserts.assert_greater_equal(ccdid, 0, "Invalid CCDID returned")
         asserts.assert_less_equal(ccdid, 65534, "Invalid CCDID returned")
 
-    def assert_valid_csr(
```

Comment:
This should've been pulled out to a common utility instead of copy-pasted

Reply:
Tracked for follow-up @ https://github.com/project-chip/connectedhomeip/issues/40931